PortMaster/utils/patcher.txt

```diff
@@ -1,17 +1,5 @@
 #!/bin/bash
 
-XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
-
-if [ -d "/opt/system/Tools/PortMaster/" ]; then
-  controlfolder="/opt/system/Tools/PortMaster"
-elif [ -d "/opt/tools/PortMaster/" ]; then
-  controlfolder="/opt/tools/PortMaster"
-elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
-  controlfolder="$XDG_DATA_HOME/PortMaster"
-else
-  controlfolder="/roms/ports/PortMaster"
-fi
-
 # Source PM info
 source $controlfolder/control.txt
 source $controlfolder/device_info.txt
```

Comment:
these too 😁

Reply:
I need those to get the $controlfolder, unless im doing something wrong somewhere else.

Reply:
This is run by the port script, no?

```sh
PATCHER_FILE="$GAMEDIR/patcher.txt"
PATCHER_GAME="Blag Machine"
PATCHER_TIME="about 7 years"

source "$controlfolder/utils/patcher.txt"
```